tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -0,0 +1,609 @@
+            super(topic, partition);
+            this.offset = offset;
+        }
+
+        @JsonProperty
+        public long offset() {
+            return offset;
+        }
+    }
+
+    private static class RecordSetSummary extends PartitionData {
+        private final long count;
+        private final long minOffset;
+        private final long maxOffset;
+
+        public RecordSetSummary(String topic, int partition, long count, long minOffset, long maxOffset) {
+            super(topic, partition);
+            this.count = count;
+            this.minOffset = minOffset;
+            this.maxOffset = maxOffset;
+        }
+
+        @JsonProperty
+        public long count() {
+            return count;
+        }
+
+        @JsonProperty
+        public long minOffset() {
+            return minOffset;
+        }
+
+        @JsonProperty
+        public long maxOffset() {
+            return maxOffset;
+        }
+
+    }
+
+    private static ArgumentParser argParser() {
+        ArgumentParser parser = ArgumentParsers
+                .newArgumentParser("verifiable-consumer")
+                .defaultHelp(true)
+                .description("This tool produces increasing integers to the specified topic and prints JSON metadata to stdout on each \"send\" request, making externally visible which messages have been acked and which have not.");
+
+        parser.addArgument("--topic")
+                .action(store())
+                .required(true)
+                .type(String.class)
+                .metavar("TOPIC")
+                .help("Produce messages to this topic.");
+
+        parser.addArgument("--groupId")
+                .action(store())
+                .required(true)
+                .type(String.class)
+                .metavar("GROUP_ID")
+                .help("The groupId shared among members of the consumer group");
+
+        parser.addArgument("--max-messages")
+                .action(store())
+                .required(false)
+                .type(Integer.class)
+                .setDefault(-1)
+                .metavar("MAX-MESSAGES")
+                .dest("maxMessages")
+                .help("Consume this many messages. If -1, the consumer will consume until the process is killed externally");
+
+        parser.addArgument("--session-timeout")
+                .action(store())
+                .required(false)
+                .type(Integer.class)
+                .setDefault(30000)
+                .metavar("TIMEOUT_MS")
+                .dest("sessionTimeout")
+                .help("Set the consumer's session timeout");
+
+        parser.addArgument("--verbose")
+                .action(store())
+                .required(false)
+                .setDefault(false)
+                .type(Boolean.class)
+                .metavar("VERBOSE")
+                .help("Set to false to only log rebalances and commits");
+
+        parser.addArgument("--enable-autocommit")
```

Comment:
At the moment no
However your wrapper python class can still take `auto_commit` as a constructor parameter, and instead of using it to update the start command by adding `--auto-commit`, put it in `verifiable_consumer.properties` file